app/models/manager_refresh/inventory/middleware_manager.rb

```diff
@@ -5,7 +5,7 @@ module ManagerRefresh::Inventory::MiddlewareManager
   COMMON_ATTRIBUTES = %i(name ems_ref nativeid properties feed).freeze
 
   class_methods do
-    # rubocop:disable Style/PredicateName
+    # rubocop:disable Naming/PredicateName
```

Comment:
Same for Naming

Reply:
I reverted the commit that did the rubocop changes.
Sorry, I used rubocop v0.51.0